image/image_linux.go

```diff
@@ -434,8 +436,8 @@ func setupSeed(tsto *ToolingStore, model *asserts.Model, opts *Options) error {
 			if err := os.MkdirAll(sysconfig.WritableDefaultsDir(rootDir, "/etc"), 0755); err != nil {
 				return err
 			}
-			applyOpts := &configcore.FilesystemOnlyApplyOptions{Classic: opts.Classic}
-			return configcore.FilesystemOnlyApply(defaultsDir, configcore.PlainCoreConfig(defaults), applyOpts)
```

Comment:
Yes, it was intentional (and discussed with Samuele). The general plan is:
- configcore only exports Run().
- FilesystemOnlyApply is only available through sysconfig (even if the actual impl lives in configcore)
- import cycles will happen because configcore imports devicestate, devicestate imports sysconfig, so sysconfig cannot import configcore, therefore we need hooks.

Reply:
I see thanks for explaining this, makes sense now